backend/api/experiment.proto

```diff
@@ -114,7 +116,14 @@ message GetExperimentRequest {
 }
 
 message ListExperimentsRequest {
+  // A page token to request the next page of results. The token is acquried
+  // from the nextPageToken field of the response from the previous
+  // ListExperiment call.
   string page_token = 1;
+
+  // The number of experiments to be listed per page. If there are more
+  // experiments than this number, the response message will contain a valid
+  // value in the nextPageToken field.
```

Comment:
nit: the response message will contain a nextPageToken field you can use to fetch the next page.
WDYT

Reply:
done